### PR TITLE
Create 2023-10-16-console.md

### PR DESCRIPTION
**Pull request by danieltprice**
Release note:
https://neon-next-git-dprice-10-16-console-relnotes-neondatabase.vercel.app/docs/release-notes

Autoscaling guide:
https://neon-next-git-dprice-10-16-console-relnotes-neondatabase.vercel.app/docs/guides/autoscaling-guide

Auto-suspend guide:
https://neon-next-git-dprice-10-16-console-relnotes-neondatabase.vercel.app/docs/guides/auto-suspend-guide

Various other minor updates throughout the docs.

### CODE REVIEW

File: content/docs/connect/connection-latency.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ Given the potential impact on application responsiveness, it's important to have
...
 
-The [Neon Pro plan](/docs/introduction/pro-plan) allows you to configure the period before the system scales down to zero, providing you with control over the balance between performance and cost. The configuration setting is called **Auto-suspend delay**, and it is set to 300 seconds (5 minutes) by default. You can disable Auto-suspend entirely or increase the setting up to a maximum of 7 days. This strategy can eliminate or reduce compute startup times, but also increases compute usage. For configuration instructions, see [Edit a compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
+The [Neon Pro plan](/docs/introduction/pro-plan) allows you to configure the period before the system scales down to zero, providing you with control over the balance between performance and cost. The **Suspend compute after a period of inactivity** setting is set to 5 minutes by default. You can disable Auto-suspend entirely or set a custom period up to a maximum of 7 days. This strategy can eliminate or reduce compute startup times, but also increases compute usage. For configuration instructions, see [Edit a compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
```
Comment (bgrenon) [inline content/docs/connect/connection-latency.md:45]: For me, the "inactivity" part of the equation doesn't seem clear enough. Consider:

"With the Neon Pro plan, you can configure the length of time that the system remains in an inactive state before Neon scales your compute down to zero. This lets you set the balance between performance (never scaling down) and cost (scaling to zero at reasonable intervals)."

Also consider adding some info in this sentence to make the pronoun reference clearer:

"Limiting or disabling autosuspend can eliminate or reduce startup times, but it also increases compute usage."

File: content/docs/guides/auto-suspend-guide.md [hunk 2]
```diff
@@ -7,49 +7,62 @@ updatedOn: '2023-08-09T08:55:53Z'
 
 Neon's [Auto-suspend](/docs/introduction/auto-suspend) feature controls when a Neon compute instance transitions to an `Idle` state (scales to zero) due to inactivity. This guide demonstrates how to configure the _Auto-suspend_ setting in your Neon project.
 
-You can configure the _Auto-suspend_ setting in an existing project by editing a compute endpoint. You can also configure it when you first create a Neon project, which sets the _Auto-suspend_ default for the project. Both methods are described below.
+[Neon Pro plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating compute endpoints.
```
Comment (bgrenon) [inline content/docs/guides/auto-suspend-guide.md:10]:
```suggestion
[Neon Pro plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating new compute endpoints.
```

File: content/docs/guides/auto-suspend-guide.md [hunk 3]
```diff
@@ -7,49 +7,62 @@ updatedOn: '2023-08-09T08:55:53Z'
...
 
-## Configure Auto-suspend for a compute endpoint
+### Configure the Auto-suspend setting for a new project
 
-[Neon Pro plan](/docs/introduction/pro-plan) users can edit an individual compute endpoint to adjust the _Auto-suspend_ configuration.
+Configuring the _Auto-suspend_ setting for a new project sets the project's default, which is applied to all compute endpoints created from that point forward. You can adjust _Auto-suspend_ default or configure the setting for individual compute endpoints later, as necessary.
```
Comment (bgrenon) [inline content/docs/guides/auto-suspend-guide.md:14]:
```suggestion
Configuring the _Auto-suspend_ setting for a new project sets the project's default, which is applied to all compute endpoints created from that point forward. You can adjust this _Auto-suspend_ default at any time, or configure the setting for individual compute endpoints later, as necessary.
```

File: content/docs/guides/auto-suspend-guide.md [hunk 4]
```diff
@@ -7,49 +7,62 @@ updatedOn: '2023-08-09T08:55:53Z'
...
     </Admonition>
 
 1. Click **Save**.
+1. Click **Create Project**. Your initial compute endpoint is created with the specified setting.
 
-## Set the Auto-suspend default when creating a project
+### Configure the Auto-suspend setting for an existing project
 
-A project is a top-level object in the Neon object hierarchy. You can think of it as a container for all other objects, including branches and compute endpoints.
+Configuring the _Auto-suspend_ setting for an existing project sets the project's default, which is applied to all compute endpoints created from that point forward. Existing compute endpoints are unaffected. You can adjust _Auto-suspend_ default or configure the setting for individual compute endpoints later, as necessary.
```
Comment (bgrenon) [inline content/docs/guides/auto-suspend-guide.md:32]:
```suggestion
Configuring the _Auto-suspend_ setting for an existing project sets the project's default, which is applied to all compute endpoints created from that point forward. Existing compute endpoints are unaffected. You can adjust the _Auto-suspend_ default or configure the setting for individual compute endpoints later, as necessary.
```

File: content/docs/guides/autoscaling-guide.md [hunk 5]
```diff
@@ -1,15 +1,52 @@
...
+_Autoscaling_ is a [Neon Pro plan](/docs/introduction/pro-plan) feature. Pro plan users can enable Autoscaling for a new project, for an existing project, or for an individual compute endpoint. _Autoscaling_ is supported with both read-write and read-only compute endpoints. Read-only compute endpoints enable Neon's [Read replica](/docs/introduction/read-replicas) feature.
 
-You can enable _Autoscaling_ in an existing project by editing a compute endpoint. You can also enable it when you first create a Neon project, which sets the _Autoscaling_ defaults for your project. Both methods are described below.
+## Enable Autoscaling for a new project
+
+Enabling Autoscaling when you create a project allows you to set _Autoscaling_ default settings for all compute endpoints created in your project. You can adjust _Autoscaling_ settings for individual compute endpoints afterward, but setting defaults when creating a project saves you from having to configure the settings for each compute endpoint later on.
+
+To enable Autoscaling at project creation time:
```
Comment (bgrenon) [inline content/docs/guides/autoscaling-guide.md:17]:
```suggestion
To enable Autoscaling when you first create your project:
```

File: content/docs/guides/auto-suspend-guide.md [hunk 6]
```diff
@@ -7,49 +7,62 @@ updatedOn: '2023-08-09T08:55:53Z'
...
+[Neon Pro plan](/docs/introduction/pro-plan) users can configure the _Auto-suspend_ setting for a new project, for an existing project, or for an individual compute endpoint. Configuring the _Auto-suspend_ setting for a new or existing project sets the project's default Auto-suspend setting, which is used from that point forward when creating compute endpoints.
 
-## Configure Auto-suspend for a compute endpoint
+### Configure the Auto-suspend setting for a new project
 
-[Neon Pro plan](/docs/introduction/pro-plan) users can edit an individual compute endpoint to adjust the _Auto-suspend_ configuration.
+Configuring the _Auto-suspend_ setting for a new project sets the project's default, which is applied to all compute endpoints created from that point forward. You can adjust _Auto-suspend_ default or configure the setting for individual compute endpoints later, as necessary.
 
-To edit a compute endpoint:
+To configure the _Auto-suspend_ default setting at project creation time:
```
Comment (bgrenon) [inline content/docs/guides/auto-suspend-guide.md:16]: Consider changing "at project creation time" to "when you first create your project"

File: content/docs/guides/autoscaling-guide.md [hunk 7]
```diff
@@ -1,15 +1,52 @@
...
+3. Specify a name, a Postgres version, and a region.
+4. Under **Compute size**, select the **Autoscaling** option.
+5. Using the slider, specify a minimum and maximum compute size.
+    ![Autoscaling](/docs/guides/autoscaling_project_creation.png)
+
+    <Admonition type="note">
+    You can configure the Auto-suspend setting for your compute endpoint at the same time. The **Suspend compute after a period of inactivity** setting defines the period of inactivity after which a compute endpoint is automatically suspended. This feature is also referred to as "scale-to-zero".
+    </Admonition>
+
+6. Click **Create Project**. Your initial compute endpoint is created with the specified settings. All compute endpoints created in the future when creating a branch or adding a compute endpoint to a branch are created with this setting.
```
Comment (bgrenon) [inline content/docs/guides/autoscaling-guide.md:30]: Consider simplifying to just:

"All future compute endpoints that you create will use this setting"

File: content/docs/connect/connection-latency.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/connect/connection-latency.md:45]: Fixed. Thank you.

File: content/docs/guides/auto-suspend-guide.md (hunk 6 above)
Comment (danieltprice) [reply content/docs/guides/auto-suspend-guide.md:17]: Fixed. TY.

File: content/docs/guides/autoscaling-guide.md (hunk 7 above)
Comment (danieltprice) [reply content/docs/guides/autoscaling-guide.md:30]: Fixed. TY.

**Review by duskpoet — COMMENTED** [review]
Release notes look good!

**Review by PlayLikeNeverB4 — APPROVED** [review]

File: content/docs/guides/read-replica-data-analysis.md [hunk 8]
```diff
@@ -40,7 +40,7 @@ Creating a read replica involves adding a read-only compute endpoint to a branch
 2. Select the branch where your database resides.
 3. Click **Add compute**.
 4. On the **Create Compute Endpoint** dialog, select **Read-only** as the **Compute type**.
-5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Auto-suspend delay** period, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 300 seconds (5 minutes). You can set this value up to 604800 seconds (7 days).
+5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Suspend compute after a period of inactivity** setting, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 5 minutes. You can set this value up to 7 days.
```
Comment (bgrenon) [inline content/docs/guides/read-replica-data-analysis.md:43]: "idle time" and "idle state" could be thought of as synonymous. Maybe use your other wording: "...after which your compute is automatically suspended" or "..after which your compute is scaled down to zero".

Should we also think about trying to get the billing implications (i.e., the benefit) in there, or is that obvious enough?  Something like: 

"...after which your compute is automatically scaled down to zero (no further billing charges until the compute restarts)."

Thoughts?

File: content/docs/guides/read-replica-guide.md [hunk 9]
```diff
@@ -33,7 +33,7 @@ Creating a read replica involves adding a read-only compute endpoint to a branch
 2. Select the branch where your database resides.
 3. Click **Add compute**.
 4. On the **Create Compute Endpoint** dialog, select **Read-only** as the **Compute type**.
-5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Auto-suspend delay** period, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 300 seconds (5 minutes). You can set this value up to 604800 seconds (7 days).
+5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Suspend compute after a period of inactivity** setting, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 300 seconds (5 minutes). You can set this value up to 604800 seconds (7 days).
```
Comment (bgrenon) [inline content/docs/guides/read-replica-guide.md:36]: Same here: "idle time" and "idle state" could be misconstrued as the same thing.

File: content/docs/guides/read-replica-prisma.md [hunk 10]
```diff
@@ -26,7 +26,7 @@ You can create one or more read replicas for any branch in your Neon project. Cr
 2. Select the branch where your database resides.
 3. Click **Add compute**.
 4. On the **Create Compute Endpoint** dialog, select **Read-only** as the **Compute type**.
-5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Auto-suspend delay** period, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 300 seconds (5 minutes). You can set this value up to 604800 seconds (7 days).
+5. Specify the **Compute size** options. You can configure a **Fixed Size** compute with a specific amount of vCPU and RAM (the default) or enable **Autoscaling** and configure a minimum and maximum compute size. You can also configure the **Suspend compute after a period of inactivity** setting, which is the amount of idle time after which your read-only compute is transitioned to an idle state. The default setting is 5 minutes. You can set this value up to 7 days.
```
Comment (bgrenon) [inline content/docs/guides/read-replica-prisma.md:29]: "idle state"

File: content/docs/introduction/compute-lifecycle.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ updatedOn: '2023-08-30T09:50:46Z'
...
+`Active` means that Postgres is currently running. If there are no active queries for 5 minutes, your compute node is automatically placed into an `Idle` state to save on energy and resources. Neon [Pro plan](/docs/introduction/pro-plan) users can disable this auto-suspension behavior so that a compute always remains active, or they can increase or decrease the amount of time after which a compute is placed into an `Idle` state. Auto-suspension behavior is controlled by an **Suspend compute after a period of inactivity** setting. For information about configuring this setting, see [Edit a compute endpoint](/docs/manage/endpoints#edit-a-compute-endpoint).
```
Comment (bgrenon) [inline content/docs/introduction/compute-lifecycle.md:11]: This paragraph defines "idle" state nicely. It is not "idle time" exactly that leads to an idle state, but "no active queries". 

I'll re-look at previous comments based on this.

File: content/docs/introduction/pro-plan.md [hunk 12]
```diff
@@ -31,7 +31,7 @@ The Neon Pro plan comes with the following advanced features, and we plan to add
...
+- **Configurable Auto-suspend:** The Free Tier automatically suspends computes after 5 minutes of inactivity. With the Pro plan, you can increase or decrease this time period, or you can disable the Auto-suspend feature entirely. A shorter delay, such as 60 seconds, can potentially reduce compute costs. A longer delay keeps your compute active for longer intervals, reducing the number of cold starts. To prevent cold starts entirely, in cases where even a few seconds delay is too much, you can disable the Auto-suspend feature entirely so that your compute is always active. The Auto-suspend feature is configured in the Neon Console. For instructions, see [Auto-suspend configuration](/docs/manage/endpoints#auto-suspend-configuration).
```
Comment (bgrenon) [inline content/docs/introduction/pro-plan.md:34]: I like the context/benefit of the feature described here. Well-captured.

**Review by bgrenon — APPROVED** [review]
Minor notes. Looks good!

File: content/docs/guides/read-replica-data-analysis.md (hunk 8 above)
Comment (danieltprice) [reply content/docs/guides/read-replica-data-analysis.md:43]: Thank you. Fixed.

File: content/docs/guides/read-replica-guide.md (hunk 9 above)
Comment (danieltprice) [reply content/docs/guides/read-replica-guide.md:36]: Fixed.

File: content/release-notes/2023-10-17-console.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+### Fixes & improvements
+
+- UI: Added new fields for configuring the auto-suspend delay period for Neon compute instances, allowing you to specify the delay period in seconds, minutes, hours, or days.
+
+    ![Auto-suspend delay controls](/docs/relnotes/autosuspend_controls.png)
+
+    Neon's _Auto-suspend_ feature automatically transitions a compute endpoint into an `Idle` state after a period of inactivity, also known as "scale-to-zero". By default, suspension occurs after 5 minutes of inactivity. [Neon Pro Plan](/docs/introduction/pro-plan) users can configure the auto-suspend setting when creating a project, changing default compute settings, creating a compute endpoint, or editing a compute endpoint.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/release-notes/2023-10-17-console.md]: Added! TY

File: content/docs/guides/read-replica-prisma.md (hunk 10 above)
Comment (danieltprice) [reply content/docs/guides/read-replica-prisma.md]: Fixed. TY.

File: content/docs/introduction/pro-plan.md (hunk 12 above)
Comment (danieltprice) [reply content/docs/introduction/pro-plan.md:34]: Perhaps I can circle back and update similar content with this context/benefit info.

**Pull request merged**